### DISCUSSION TITLE
how to use sense2vec for dataset other languages?

### DISCUSSION THREAD

**Discussion by clovisholanda** [explosion/spaCy#10018] [Help: Other Questions] [answered]
Hey guys,

I'm a beginner, please help me,
I would like code examples on how to use sense2vec with datasets for the Portuguese language
I just found an example in the English language

Example web:

import spacy
nlp = spacy.load("en_core_web_sm")
**#nlp = spacy.load("pt_core_news_sm") # portugues**
s2v = nlp.add_pipe("sense2vec")
s2v.from_disk("s2v_reddit_2015_md/s2v_old") # **put a dataset em portugues**
...

Clovisholanda

**Comment by polm** [comment, marked as answer]
sense2vec was designed with English in mind. To use it with Portuguese you'd need to adapt the rules (probably not too hard) and train your own model using a large corpus. Reddit is basically just English so maybe you could use Wikipedia or something.

Unfortunately we don't have a guide for this.

**Reply by clovisholanda** [reply]
Thank you very much!!!! :)

**Reply by jozgu** [reply]
@clovisholanda did you manage to build a portugues model? 
If so, how was your approach. Trying to figure out a way to do it for a danish setup

**Reply by polm** [reply]
If you have a specific question, feel free to open a new thread where we can try to help you.
